advanced-menu/advanced-menu.py

```diff
@@ -645,12 +674,13 @@ async def advancedmenu_load_config(self, ctx):
         if not ctx.message.attachments:
             return await ctx.send("You must attach a json file to load the config from.")
         b = await ctx.message.attachments[0].read()
-        json_data = b.decode("utf-8").replace("'", '"')
+        json_data = b.decode("utf-8")
```

Comment:
Why remove the .replace()
And why do this in the same PR as a feature update?

Reply:
I removed this because I could not load my exported configs due to my messages in strings containing a ' in them. This replace would cause them to turn into " and invalidate the JSON. I wrote more details in a comment to the PR.